src/lib/config.ts

```diff
@@ -238,7 +239,6 @@ class default_config {
         ";gv": "hint -qW exclaim_quiet mpv",
         ";gw": "hint -qw",
         ";gb": "hint -qb",
-        "<S-Insert>": "mode ignore",
```

Comment:
I'd like to keep this bind as it is from Vimperator. I think it would confuse people if it suddenly disappeared.